tests/lava/lib/dl/slayer/neuron/dynamics/test_leaky_integrator.py

```diff
@@ -9,6 +9,9 @@
 import torch.nn.functional as F
 import lava.lib.dl.slayer as slayer
 
+print(slayer.__path__)
```

Comment:
Have not reviewed in detail...
But this looks like a strange unit test. Why is there so much low level code outside of test cases?

Also unit tests must be silent and not print anything by default

Reply:
I think you are looking at an old verison. It's been removed